packages/api-v2/src/hooks/useGrowthbookIsOn.ts

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from 'react';
+import { Analytics } from '@deriv-com/analytics';
+import useIsGrowthbookIsLoaded from './useIsGrowthbookLoaded';
+
+interface UseGrowthbookIsOneArgs {
+    featureFlag: string;
+}
+
+const useGrowthbookIsOn = ({ featureFlag }: UseGrowthbookIsOneArgs) => {
```

Comment:
this already exists in the hooks package, what do we need it here?

Reply:
yes @deriv/hooks is not be used in wallets. As @deriv/hooks uses stores. So wallet team maintains a copy of all hooks in api-v2. This was added to api-v2 as per Nijil